source/onboard/shared-channels.rst

```diff
@@ -1,40 +1,38 @@
-Shared channels (experimental)
-==============================
+Shared channels (Beta)
```

Comment:
Shared channels is still going to stay "Experimental" until we release the UI being worked on right now